test/services/BaseSupabaseServices.test.ts

```diff
@@ -0,0 +1,121 @@
+import { Kysely } from "kysely";
```

Comment:
I wonder how useful these tests are. They're essentially just testing that the service is passing the args along.